### DISCUSSION TITLE
"Generating SSL Certificate" Stuck

### DISCUSSION THREAD

**Discussion by toledoroy** [vercel/community#2806] [Help] [answered]
### Summary

Hi,
Added a new domain and it's been stuck like that for days.
Anything I can do to fix this?

### Steps to Reproduce

<img width="671" alt="image" src="https://github.com/vercel/community/assets/5618090/d917e4ba-72da-4df8-af79-4fa6d2a14f08">

**Comment by smaeda-ks** [comment, marked as answer]
```
$ dig soulsystem.app @1.1.1.1

; <<>> DiG 9.18.12 <<>> soulsystem.app @1.1.1.1
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: SERVFAIL, id: 26916
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
; EDE: 9 (DNSKEY Missing): (no SEP matching the DS found for soulsystem.app.)
;; QUESTION SECTION:
;soulsystem.app.			IN	A

;; Query time: 17 msec
;; SERVER: 1.1.1.1#53(1.1.1.1) (UDP)
;; WHEN: Fri Jun 09 19:43:52 JST 2023
;; MSG SIZE  rcvd: 97
```

> ; EDE: 9 (DNSKEY Missing): (no SEP matching the DS found for soulsystem.app.)

You have some misconfiguration on your domain. Could you double-check that you disabled DNSSEC in your domain registrar? We don't support DNSSEC as of today.

**Comment by toledoroy** [comment]
Gotcha. Thanks!
